create_tables.sql

```diff
@@ -0,0 +1,42 @@
+DROP TABLE IF EXISTS logs CASCADE;
+DROP TABLE IF EXISTS stats CASCADE;
+DROP TABLE IF EXISTS workers CASCADE;
+DROP TABLE IF EXISTS components CASCADE;
+DROP TABLE IF EXISTS users CASCADE;
+
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY,
+  email VARCHAR(100) NOT NULL,
```

Comment:
Please use `TEXT` instead of `VARCHAR` -- it's just as efficient, and means we won't get stupid db errors -- that goes for all current `VARCHAR` fields.